.github/workflows/python-tests.yml

```diff
@@ -25,14 +25,20 @@ jobs:
         - 3.6
         - 3.5
         os:
-        - ubuntu-latest
+        - ubuntu-18.04
```

Comment:
This change comes with an implicit TODO to keep it up-to-date (adds debt to the config). I'd rather track the latest and bump 16.04 to 18.04. WDYT?

Reply:
If you change 18.04, you'll have it double (18.04+latest would be the same) until the latest doesn't switch to 20.04. This change will be silent and will result in breaking expectations. So there will be the debt to notice when the latest changes and switch other entries on time...